### ISSUE TITLE
Possible enhancement to how designer connects activities

### ISSUE THREAD

**Issue by tomy2105** [elsa-workflows/elsa-core#2405]
Currently when multiple activities outcomes connect to single activity all the connection lines are kept separate which adds to "noise" on the screen, especially in the cases when activities are far apart and connecting lines are long.

I think it would be better if lines were "merged" into one as soon as possible (provided they are relatively close to each other).
This is "partially" connected to #2404, when outcome names and + icons are placed near the "source" activity the possibility to merge lines will become even more evident, I think.

What do others think?

**Comment by sfmskywalker** [comment]
Makes sense. Should be fixed when we backport the Elsa 3 designer that's currently under development.

**Comment by sfmskywalker** [comment]
Closing this as stale because it targets the legacy Elsa 2 dashboard/designer surface that used to live in `elsa-core` (the issue body/title references the old designer, instance viewer, or Elsa 2.x preview builds). That UI surface is no longer an active subsystem in this repository, so we are not planning more work on these backlog items here. If the same problem still exists in the current Studio experience, please open a fresh issue in `elsa-workflows/elsa-studio` with current version details and repro steps.
